src/laser_geometry.cpp

```diff
@@ -302,6 +302,8 @@ const boost::numeric::ublas::matrix<double>& LaserProjection::getUnitVectors_(do
       {
         ranges(i,0) = (double) (scan_in.ranges[i] > range_cutoff
                           ? range_cutoff : scan_in.ranges[i]);
+        ranges(i,0) = (double) (scan_in.ranges[i] < scan_in.range_min
```

Comment:
You could do:

`ranges(i,0) = (double) (scan_in.ranges[i] < scan_in.range_min ? scan_in.range_min : ranges(i,0)`

or better:
```
if (scan_in.ranges[i] > range_cutoff)
{
  ranges(i,0) = range_cutoff;
}
else if (scan_in.ranges[i] < scan_in.range_min)
{
  ranges(i,0) = scan_in.range_min;
}
else
{
  ranges(i,0) = scan_in.ranges[i];
}
```